packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1077,8 +1083,6 @@ export default class Meeting extends StatelessWebexPlugin {
     this.locusInfo.init(attrs.locus ? attrs.locus : {});
     this.hasJoinedOnce = false;
 
-    this.media = new MultistreamMedia(this);
```

Comment:
I've removed the separate MultistreamMedia class and moved the publish/unpublish methods back into meeting/index.ts. The only reason why we had them in the separate class was so that it could be written in typescript, now the whole Meeting object is typescript already.